src/pydata_sphinx_theme/assets/styles/abstracts/_links.scss

```diff
@@ -164,114 +179,54 @@ $link-hover-decoration-thickness: string.unquote(
   }
 }
 
-// Navigation bar current page link styles
-// ---------------------------------------
-// Adds a bottom underline, this leaves enough space for the hover state without
-// cluttering the navbar.
-// We want the side box shadow to have the same thickness as the hover underline
-@mixin link-navbar-current {
```

Comment:
Deleted this mixin because I couldn't find it being used anywhere in the codebase